content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,570 @@
+# Running Experiments
+
+## Motivation
+
+Machine Learning and Data Science projects usually involve experimentation.
+These experiments' goals can range from finding good hyperparameters to
+testing for data and concept drift. DVC 2 introduced a new set of commands to
+manage experiments with minimum boilerplate. It allows to run experiments
+defined by pipelines, track their associated data and model files, set
+parameters for each, push experiment parameters and code to Git remotes without
+committing them, create branches and persist them in Git. 
+
+In this part of the DVC User's Guide we explain how DVC runs the experiments,
+set the parameters, use multiple jobs to run the experiments in parallel, or
+running them in temporary directories. 
+
+If this is the first time you are introduced into Machine Learning
+experimentation, you may find a quicker introduction to the most salient
+features in the [Experiment's Trail](/doc/start/experiments/).
+
+## The Pipeline
+
+DVC uses pipelines composed of <abbr>stages</abbr>. Stages are granular and
+interdependent steps that has a command, a set of dependencies, and output.
+Stages can depend onto each other via their outputs that build a <abbr>DAG</abbr>.
+
+DVC employs the pipeline to run the experiment commands. Each pipeline has an
+end-point, which is the last command in the pipeline. DVC also tracks artifacts
+from the intermediate stages, and doesn't reproduce them if their dependents
+have not changed.
+
+DVC pipelines are a detailed topic that we cover in [Get Started][gs-pipelines]
+and the [User's Guide][ug-pipeline-files]. Here we assume that there is already
+a pipeline defined in `dvc.yaml` file in the <abbr>project</abbr>.
+
+[gs-pipelines]: /doc/start/data-pipelines
+[ug-pipeline-files]: /doc/user-guide/project-structure/pipelines-files
+
+### Running the pipeline
+
+You can run the pipeline defined in `dvc.yaml` file with the default settings
+using:
+
+```dvc
+$ dvc exp run
+```
+
+If there are no missing or changed outputs in the the workspace, the `dvc exp
+run` doesn't rerun the commands. DVC keeps track of the dependency graph and
+runs only the stages with missing or changed dependencies. 
+
+> Example: For a pipeline composed of `extract`, `transform`, `train`,
+> `evaluate`, if a dependency of `train` stage has changed, the dependent
+> stages (`evaluate`) are also run.
+
+#### Running Specific Stages
+
+By default `dvc exp run` uses `dvc.yaml` file in the current directory. You can
+specify other directories or pipeline elements to run. These are specified as
+the last element of the command.
+
+Pipelines can be of the following:
+
+- A particular `dvc.yaml` file: `dvc exp run my-project/dvc.yaml`
+- A stage from the default `dvc.yaml` file: `dvc exp run extract`
+- A stage from a specific `dvc.yaml` file:
+  `dvc exp run my-project/dvc.yaml:extract`
+- A set of stages from the default `dvc.yaml` file:
```

Comment:
I'm removing  `--recursive` and `--glob` from these docs until iterative/dvc#6456 and iterative/dvc#6457 are resolved.

Reply:
Opened an issue to restore them in #2730

Reply:
I think it's OK to document the intended behavior but OK up to you.

Reply:
We can't have a timeframe for the implementation. I'd remove these options from the command reference as well.

Reply:
> I'd remove these options from the command reference as well.

Probably not necessary (haven't noticed many support cases about it at least) but feel free to!

Resolving